packages/darklang/languageTools/writtenTypes.dark

```diff
@@ -153,6 +154,12 @@ module Darklang =
           symbolOpenQuote: SourceRange *
           symbolCloseQuote: SourceRange
 
+        | EList of
+          SourceRange *
+          contents: Stdlib.Option.Option<SourceRange * List<Expr>> *
```

Comment:
I think a better 'shape' for this might be
```suggestion
          contents: List<Expr * Option<SourceRange>> *
```
where:
- the Expr is ... just the expr, which already contains its own SourceRange
- the SourceRange after that is where the `;` exists, if it does

Some notes:
- the 'optionalness' of the content is implied by contents just being empty
- this also accounts for `[1; 2;]` noted elsewhere, which I think is a realistic scenario
- this set-up is a bit less nested, and might be easier to work with